src/app/containers/Video/index.stories.jsx

```diff
@@ -1,12 +1,36 @@
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
-import { VideoWithCaption, VideoWithoutCaption } from './fixtureData';
+import {
+  AudioGlobalGuidance,
+  VideoWithCaption,
+  VideoWithoutCaption,
+  VideoPortrait,
+  VideoWithGuidanceUk,
+  VideoNonUk,
+  AudioUkNoGuidance,
+  AudioNonUkNoGuidance,
+  AudioEpisodeGlobal,
+} from './fixtureData';
 import AmpDecorator from '../../helpers/storybook/ampDecorator';
 
 storiesOf('Video Container', module)
-  .add('with a caption', () => VideoWithCaption)
-  .add('without a caption', () => VideoWithoutCaption);
+  .add('[video] with caption', () => VideoWithCaption)
+  .add('[video] without a caption', () => VideoWithoutCaption)
+  .add('[video] portrait', () => VideoPortrait)
+  .add('[video] UK with guidance', () => VideoWithGuidanceUk)
+  .add('[video] non-UK', () => VideoNonUk)
+  .add('[audio] global with guidance', () => AudioGlobalGuidance)
+  .add('[audio] UK without guidance', () => AudioUkNoGuidance)
+  .add('[audio] non-UK without guidance', () => AudioNonUkNoGuidance)
+  .add('[audio] episode, global', () => AudioEpisodeGlobal);
```

Comment:
It'd be good to have consistency in the naming of these. Both in the summary and in the name of the component.
E.g. `video clip, global, with guidance` and not assume that clip is the default, etc.